superset/assets/src/dashboard/actions/dashboardState.js

```diff
@@ -135,6 +135,23 @@ export function onSave() {
   return { type: ON_SAVE };
 }
 
+export const ON_RECONCILE = 'ON_RECONCILE';
+export function onReconcile() {
+  return { type: ON_RECONCILE };
+}
+
+export const ON_FINISH_RECONCILE = 'ON_FINISH_RECONCILE';
+export function onFinishReconcile(publishSubscriberMap) {
+  return { type: ON_FINISH_RECONCILE ,publishSubscriberMap};
+}
+
+export function reconcileSuccess(data,id){
+  return dispatch => {
+    dispatch(onFinishReconcile(data.pub_sub_info))
+    dispatch(saveDashboardRequest(data,id,SAVE_TYPE_OVERWRITE,'This dashboard was reconcile successfully.'));
```

Comment:
keep space after ever common to remove lint errors.

Reply:
done